tee-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -143,6 +160,52 @@ where
 		Ok(json!(json_value))
 	});
 
+	if let Some(state) = state {
+		// state_getStorage
+		let state_get_storage = "state_getStorage";
+		io.add_sync_method(state_get_storage, move |params: Params| {
+			// jsonrpc_core::serde::Deserialize<'de>` is not implemented for `H256`
+			// can not parse ShardIdentifier
+			// TODO: hanld error
```

Comment:
should fix the `TODO`, just create another function `invalid_params` to return the error response.